src/Model/PatientDictContainer.py

```diff
@@ -7,25 +12,60 @@
     num_points
     pixluts
 """
+from src.Model.Singleton import Singleton
+
 
+class PatientDictContainer(metaclass=Singleton):
 
-class PatientDictContainer:
+    def __init__(self):
+        # Initialize base requirements
+        self.path = None                # The path of the loaded directory.
+        self.dataset = None             # Dictionary of PyDicom dataset objects.
```

Comment:
suggest using type hints.
And the name is problematic.
pydicom.dataset.Dataset is the underlying type, and it's referred to as a dataset.
Here, dataset means what the comment says (so at least it's commented, but it wouldn't hurt to use type hints, e.g

from typing import Dict
...
self.dataset : Dict = None
or alternatively to use comment based type hinting